src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -1015,6 +1015,10 @@ protected function addArrayAccessorBody(&$script, Column $column)
      */
     protected function addBooleanAccessor(&$script, Column $column)
     {
+        $name = $column->getCamelCaseName();
+        if (in_array($name, ClassTools::getPropelReservedMethods())) {
+            return; // Skip boolean accessors for reserved names
+        }
```

Comment:
We do not have a proper way of issuing warnings now in the builder, only throwing errors. I do believe that boolean accessors are not mandatory and a warning can be issued suggesting the user to use the normal accessor.